### PR TITLE
Working

### PR DESCRIPTION
**Pull request by kelibst**
> A basic JavaScript To do app for creating tasks and adding to do's.
Read more on the project specifications on [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list)

- Loads Contents directly to the page using Javascript.
- Users can add Task. Todos are grouped under tasks
- Responsive

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+      - 'dist/*'
+
+# PLEASE DO NOT enable auto fixing options
+# if you need extra support from you linter - do it in your local env as described in README for this config
+
+# find full documentation here: https://stickler-ci.com/docs
```
Comment (Torres-ssf) [inline .stickler.yml:15]: - [ ] You can delete the stickler.yml file. You are already using Github actions, so using the stickler is not necessary.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi team :hand: 
Nice job so far in this project :clap:, I really liked your UI 
But we need to make some changes before moving on.

- [ ] The user should be able to edit the details of a to-do item. Right now you application does not have this feature.

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Changes Required❌

# 🙋‍♂️ Team 

* [ ]  I have tried to use your application locally and by using the two live links you provided in the readme. 
- [ Here is a link to the video of my findings](https://www.loom.com/share/60600cde169c46709dfc279c5f7c149b).
### Kindly make, make the address the issue and submit a re-review🧒.

#### Happy coding👨‍💻

Addo Forison

**Review by Ispirett — APPROVED** [review]
# Status Approved :+1: 
Great work! folks with all the changes made and on completing this project. :+1: 

You can change the add task button to add projects.

**Pull request merged**